.github/workflows/pipeline-golang-app.yml

```diff
@@ -82,29 +82,18 @@ jobs:
       coverageThreasholdPackage: ${{ inputs.coverageThreasholdPackage }}
       coverageThreasholdTotal: ${{ inputs.coverageThreasholdTotal }}
 
-  dockerBuildAndPushAMD:
+  dockerBuildAndPush:
     needs: [createVersion]
-    uses: ./.github/workflows/job-docker-build-push-arch.yml
+    uses: ./.github/workflows/job-docker-build-push.yml
     with:
       imageTagName: ${{ inputs.imageTagName }}
       version: ${{ needs.createVersion.outputs.version }}
       release_branch: ${{ inputs.release_branch }}
-      arch: 'linux/amd64'
-    secrets: inherit
-
-  dockerBuildAndPushARM:
-    needs: [createVersion]
-    uses: ./.github/workflows/job-docker-build-push-arch.yml
-    if: ${{ github.ref == format('refs/heads/{0}',inputs.release_branch) }}
-    with:
-      imageTagName: ${{ inputs.imageTagName }}
-      version: ${{ needs.createVersion.outputs.version }}
-      release_branch: ${{ inputs.release_branch }}
-      arch: 'linux/arm64'
+      multiarch: true
     secrets: inherit
 
   pushGithubTag:
```

Comment:
so this pushGithubTag step was only needed because of the split. 

The original job was also pushing the git tag